roles/command_line_tools/meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Install Xcode Command Line Tools
   company:
   license: Apache 2.0
-  min_ansible_version: 2.9
+  min_ansible_version: 2.11
```

Comment:
We still use 2.10 in ansible-core-ci. Technically, this bump isn't really necessary since the patch doesn't depend on any 2.11+ features IIUC.
Even though invoking playbooks from collections is only possible with newer ansible-core, I don't think that the collection metadata should take away the ability to make use of the improved roles under older ansible-core distributions.
Point is that adding features unsupported under certain user's runtime is fine if they are not core features but are rather optional — this is something that the end-users would decide whether to use.